examples/c/38_delta_system_time/src/main.c

```diff
@@ -1,15 +1,15 @@
 #include <delta_system_time.h>
 
 void Period05(ecs_iter_t *it) {
-    printf("t = 0.5, elapsed time = %f\n", it->delta_system_time);
+    printf("t = 0.5, elapsed time = %f\n", (double)it->delta_system_time);
```

Comment:
What is the point of this cast?

Reply:
I added a double-promotion warning to the build which ensures that you can't implicitly upcast from a float to a double. I use the examples to verify that you can use the API without warnings, but that means that they also have to adhere to the strictest warning settings.

It's a bit of an annoying warning, because it will trigger each time you try to pass a float to printf, but still that's better than having flecs compile with warnings.